CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ scheme are considered to be bugs.
 
 ### Miscellaneous
 
-* Your contribution here.
+* [#521](https://github.com/hashie/hashie/pull/499): Fixed typo in module name for `Hashie::Extensions::Mash::SymbolizeKeys` example documentation. - [@carolineartz](https://github.com/carolineartz)
```

Comment:
You have an extra period in here, so the bot complains. It goes to the end of the line.